### PR TITLE
collect node serial logs

### PR DESCRIPTION
**Pull request by BenTheElder**
pretty small change, in particular this lets us collect the systemd init output.

### CODE REVIEW

File: pkg/cluster/logs/logs.go [hunk 1]
```diff
@@ -87,6 +87,10 @@ func Collect(nodes []nodes.Node, dir string) error {
 					filepath.Join(name, "inspect.json"),
 				),
 				// grab all of the node logs
+				execToPathFn(
+					exec.Command("docker", "logs", name),
```
Comment (aojea) [inline pkg/cluster/logs/logs.go]: should we add `-t` to the command to have the timestamps?
I find them very useful

**Comment by BenTheElder** [comment]
I think we want consistent logging across daemons, they're unlikely to be
useful in this scenario as they're all quickly after start (it's just init
messages, then nothing). The runs from this PR should have examples

*From: *Antonio Ojea <notifications@github.com>
*Date: *Fri, May 3, 2019, 23:38
*To: *kubernetes-sigs/kind
*Cc: *Benjamin Elder, Author

*@aojea* commented on this pull request.

File: pkg/cluster/logs/logs.go (hunk 1 above)
Comment (BenTheElder) [reply pkg/cluster/logs/logs.go]: https://storage.googleapis.com/kubernetes-jenkins/pr-logs/pull/sigs.k8s.io_kind/480/pull-kind-conformance-parallel-1-14/1124528321279823874/artifacts/logs/kind-control-plane/serial.log

probably not so useful. I do think we might want to tweak the init output though.

Comment (BenTheElder) [reply pkg/cluster/logs/logs.go]: also definitely need to refactor this out to pkg/container, have a bigger PR to refactor this code later.

**Pull request merged**